### PR TITLE
feat: adding parties/candidates + election start rework

### LINKED ISSUE #2: [Admin] Add all Parties/Candidates to blockchain when election starts

### LINKED ISSUE #6: [Admin] As an administrator, I want to be able to add, edit, and remove candidates

### LINKED ISSUE #7: [Admin] As an administrator, I want to be able to add, edit, and remove parties

### PR DESCRIPTION
**Pull request by Jatewo**
## Description

This PR introduces routes for creating parties and candidates as well as a rework for the election start route which now sends all candidates and parties for inclusion on the blockchain.

## Changes made
- Cleaned up unused code and comments
- Election start route sends data to blockchain
- Key generation moved to separate file
- Added route for creating candidates
- Added route for creating parties

## Notes
- Should nomination districts and constituencies also be sent to blockchain?

### CODE REVIEW

**Review by Mafusn — APPROVED** [review]
Such amazing

File: package.json [hunk 1]
```diff
@@ -43,5 +45,17 @@
         "pg": "^8.13.0",
         "sequelize": "^6.37.3",
```
Comment (MortenMunk) [inline package.json]: Remove sequelize

**Review by MortenMunk — CHANGES_REQUESTED** [review]
Kunne være nice hvis vi afinstallerede sequelize nu når vi ikke bruger det mere

ellers fint

**Review by MortenMunk — APPROVED** [review]
LGTM

**Pull request merged**